utils/build-script

```diff
@@ -1064,8 +1063,7 @@ details of the setups of other systems or automated environments.""")
         "-n", "--dry-run",
         help="print the commands that would be executed, but do not execute "
              "them",
-        action="store_true",
-        default=False)
+        action=arguments.action.optional_bool)
```

Comment:
I don't recommend to apply `optional_bool` to options that has short-name like `"-n"`,
Because, if `-ni` is given, users expect that it's equivalent to `-n -i`, but `argparse` treat that as `-n=i`.
That's why we have separate definitions for `-t` and `--test`.